packages/plugins/src/plugins/spark/constants.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  EvmNetworkIdType,
+  NetworkId,
+  networks,
+} from '@sonarwatch/portfolio-core';
+import { Address } from 'viem';
+import { LendingConfig } from '../aave/types';
+
+export const PLATFORM_ID = 'spark';
+export const ETHEREUM_NETWORK_ID = NetworkId.ethereum;
+const ETHEREUM_NETWORK = networks[ETHEREUM_NETWORK_ID];
+
+export const SPARK_YIELD_POOLS_KEY = 'spark-yield-pools';
+
+export const LENDING_TOKEN_TRACKER_ADDRESS: Address =
+  '0xf705d2b7e92b3f38e6ae7afadaa2fee110fe5914';
+
+export const YIELD_ADDRESSES: Address[] = [
+  '0xbc65ad17c5c0a2a4d159fa5a503f4992c7b545fe',
+  '0x29bd15f2c80f2807c29d2428aa835f2be1098a62',
+];
+
+export const CONVERSION_RATE_DIVISOR = 1e18;
```

Comment:
Similar constants are defined in a few location.

DRY principle would tell us to re-use the already-existing constant.

If we want a `spark`-specific one, did you consider giving it a custom name ?

```suggestion
export const SPARK_CONVERSION_RATE_DIVISOR = 1e18;
```

Reply:
Where do you suggest i put this

Reply:
I think the location is fine if we want to make this constant `spark`-specific.

The other option I can think of is using either `ethFactor` or `ethereumNetwork.native.decimals`; see https://github.com/Octav-Labs/sonarwatch-portfolio/blob/7499f1c8a346a0260051b2128c1bd82ed30e0f0f/packages/plugins/src/utils/evm/constants.ts#L4 as a reference